grunt/config/browserify.js

```diff
@@ -126,6 +126,57 @@ function wrapperify(src) {
 });
 `;
 }
+function wrapperifyAddons(src) {
+  var toReplace =
+    `function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.${this.data.standalone} = f()}}`;
+  if (src.indexOf(toReplace) === -1) {
+    throw new Error('wrapperify failed to find code to replace');
```

Comment:
may want to refer to the wrapperify instance in this error as `wrapperifyAddons`